### PR TITLE
Add dependency packages

### PR DESCRIPTION
**Pull request by JoeyJiao**
Signed-off-by: Jiangen Jiao <joeyjiaojg@163.com>

### CODE REVIEW

**Comment by linuxep** [comment]
we don't depend on sysstat and iotop as they have been integrated. thanks for your commit. but NACK.

**Comment by JoeyJiao** [comment]
Ok, understand now, the prebuilt in your git not updated, use compiled one and no such dependency now

**Comment by JoeyJiao** [comment]
It's not arm-4.9 issue, it's make clean issue. Updated Readme, please check if can merge into your repo.

File: README.md [hunk 1]
```diff
@@ -45,9 +45,11 @@ git@iZ22ngfe4n3Z:~$ /opt/deploy_lepv/lepd &
 
 ## 如何编译
 
+NOTE: Always `make clean` before make otherwise might make fail
```
Comment (linuxep) [inline README.md]: it seems we only need to make clean while switching between x86 and ARM?

File: README.md [hunk 2]
```diff
@@ -56,18 +58,25 @@ For X86（电脑需要安装libev-dev库）:
...
 root@bob-VirtualBox:~/lepd-src# make ARCH=arm 
+root@bob-VirtualBox:~/lepd-src# make ARCH=arm install
 
 ## 如何运行
 
 运行lepd需要root权限，因为系统有些proc文件无root权限无法读取。
 
+root@bob-VirtualBox:~/lepd-src# ./prebuilt-binaries/x86-lepd
+OR
+root@bob-VirtualBox:~/lepd-src# ./prebuilt-binaries/arm-lepd
```
Comment (linuxep) [inline README.md]: it seems we can't do this for ARM? for x86, it is ok.

**Pull request closed without merging**